src/lib/constants.js

```diff
@@ -1,4 +1,10 @@
+/* eslint-disable prefer-regex-literals */
 module.exports = Object.freeze({
+  UNIQUE_USERNAME: (email) => `${email.split('@')[0]}_${new Date().valueOf()}`,
```

Comment:
I added this function to a utils fille inside the services folder like this: 
```
const generateUniqeUsername = (email) => {
  return `${email.split('@')[0]}_${new Date().valueOf()}`;
};
```
and then used it in my create method. I think it's better than putting it in the constant file.